diff --git a/node_modules/gatsby-post-gen/lib/index.js b/node_modules/gatsby-post-gen/lib/index.js
index 300e662..acf92e0 100644
--- a/node_modules/gatsby-post-gen/lib/index.js
+++ b/node_modules/gatsby-post-gen/lib/index.js
@@ -13,6 +13,9 @@ const TARGET_DIR = `${cwd}${CONTENTS_DIR}`
 const IGNORE_DIR = 'images'
 const UTF_8 = 'utf8'
 const DATE_FORMAT = 'yyyy-MM-dd HH:MM:SS'
+const DATE_ARRAY = [
+  -1.5, -1, -0.5, 0, 0.5, 1, 1.5
+];
 
 const ignoreFunc = (file, stats) =>
   stats.isDirectory() && path.basename(file) == IGNORE_DIR
@@ -119,9 +122,42 @@ const fetchTitle = async category => {
   return title
 }
 
-module.exports = (async function() {
-  const date = dateFns.format(new Date(), DATE_FORMAT)
+const fetchFakeDate = async () => {
+  const originDate = new Date().getTime()
+  const { res } = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'res',
+      message: 'Do you want fake date?: ',
+      choices: ['yes', 'no'],
+    },
+  ])
+  const isNeedFake = res === 'yes'
+  if (!isNeedFake) return dateFns.format(originDate, DATE_FORMAT)
+  const { addHour } = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'addHour',
+      message: 'How much hour do u add? (-,+ number): ',
+      validate: async val => {
+        if (isNaN(Number(val))) {
+          return 'type only number with -, +'
+        }
+
+        return true
+      },
+    },
+  ])
+  const addDate = Number(addHour) * 3600000
+
+  return dateFns.format(
+    originDate + addDate,
+    DATE_FORMAT
+  )
+}
 
+module.exports = (async function () {
+  const date = await fetchFakeDate()
   log.info('📅 Create new post:', date)
   log.start('🚚 Start to process!\n')
 
